src/components/Login/LoginContainer.tsx

```diff
@@ -251,7 +256,9 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
         setIsLoading(false);
         setLoginStage("MOBILE_NUMBER");
       } catch (e) {
-        const error = new Error(`onBarCodeScanned ${e}`);
+        const error = new Error(
+          `${i18n.t("thrownErrors.onBarcodeScanned")} ${e}`
+        );
```

Comment:
This error is being caught and never thrown, so it should never be shown to user right? Looks like we are only putting this in Sentry and are showing a different alert message. If that were the case then we shouldn't be translating this?

Reply:
Do check the other "error boundary" messages and see if they are being thrown and shown to users

Reply:
i think they're being shown here in the ErrorBoundary.tsx:
`<ErrorBoundaryContent
        error={this.state.errorMessage ? error : undefined}
      />`

Reply:
Ah i see, we are throwing to let error boundary display the message. Good catch. Bringing this convo to the main Slack channel!

Reply:
Oh I just looked at this again, actually only `error.name` is shown. This still isn't shown to the users

```
// in ErrorBoundary


static getDerivedStateFromError(error: Error): State {
  return { hasError: true, errorMessage: error.name };
}
```